src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -437,19 +510,24 @@ export default class PhotoField extends React.Component {
             <ProgressBar percent={this.state.progress}/>
           </div>}
           {!isDone && this.state.src && <div className="cropper-container-outer">
-            <Cropper
-              ref="cropper"
-              ready={this.setCropBox}
-              responsive
-              src={this.state.src}
-              aspectRatio={1}
-              cropBoxMovable={false}
-              cropend={this.handleCropend}
-              toggleDragModeOnDblclick={false}
-              dragMode="move"
-              guides={false}
-              viewMode={1}
-              zoom={this.onZoom}/>
+            <div ref="container">
+              <Cropper
+                ref="cropper"
+                key={smallScreen ? 'smallScreen' : 'largeScreen'}
+                ready={this.setCropBox}
+                responsive
+                src={this.state.src}
+                aspectRatio={1}
+                cropBoxMovable={false}
+                cropend={this.handleCropend}
+                cropmove={this.handleCropMove}
+                minContainerHeight={smallScreen ? 240 : 300}
```

Comment:
For wide images, the container will use the `canvasHeight` that was calculated to display the whole image on initial load as its minimum height. Once the minimum height is set, it cannot be updated without unmounting the component. 

Forcing the component to 
- meet the 240 / 300 height values in the styling, and 
- swapping the component `key` when the layout changes 

resolves the following issues:

<img width="560" alt="screen shot 2018-02-06 at 20 53 25" src="https://user-images.githubusercontent.com/4998130/35896059-fa4ceedc-0b7f-11e8-829d-a3eacd49e854.png">
<img width="634" alt="screen shot 2018-02-06 at 19 58 07" src="https://user-images.githubusercontent.com/4998130/35896233-eace729a-0b80-11e8-9f98-8ef7963b1b4b.png">

Unmounting the component is janky but it only happens when the threshold is crossed.